com.unity.ml-agents/Runtime/Sensors/StackingSensor.cs

```diff
@@ -221,6 +221,14 @@ internal byte[] CreateEmptyPNG()
             int height = m_WrappedSensor.GetObservationShape()[0];
             int width = m_WrappedSensor.GetObservationShape()[1];
             var texture2D = new Texture2D(width, height, TextureFormat.RGB24, false);
+            for (int h = 0; h < texture2D.height; h++)
+            {
+                for (int w = 0; w < texture2D.width; w++)
+                {
+                    texture2D.SetPixel(w, h, Color.black);
```

Comment:
I believe it's bad practice to call SetPixel repeatedly like this.

Maybe something like this?
https://answers.unity.com/questions/635383/how-can-i-change-all-of-a-textures-pixels-to-one-c.html